igcollect/ping.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+# this script is used to collect ping data to evaluate availability and latency
+# of our network (this should replace smokeping in the longrun)
```

Comment:
`(this should replace smokeping in the longrun` is interesting for the Pull Request itself but not for the script description.